### ISSUE TITLE
Repeated content 

### ISSUE THREAD

**Issue by skube** [brucemcpherson/SlidesMerge#11]
I'm wondering if it's possible to use SlidesMerge to create a series of stickies (e.g. 
 https://learninginhand.com/blog/print-custom-sticky-notes-with-google-slides)

**Comment by brucemcpherson** [comment]
Yes .. any text or images can be positioned with a placeholder in the template and populated from a sheet, one slide per row is generated.

**Comment by skube** [comment]
Yes, but If I wanted _multiple_ rows per slide, I can't seem to do it. I don't want to use one full 8.5x11" Slide just for each 3x3" stickie. 

I was asking if it's possible to somehow have multiple rows merged into to one slide. For example, if I had 6 rows in **Sheets**, it would create and merge only 1 **Slide** (with 6 stickies). If I had 9 rows, it would make 1 full Slide and only partially fill the second Slide, for a total of 9 stickies across two Slides.

I understand that's not the intended purpose of your Add-on, so I was wondering if there was either some trick to merging or if your add-on could easily be updated to have such feature.  😄

**Comment by brucemcpherson** [comment]
You could simply repeat the placeholder multiple times on one slide, if it's the same one repeated, or you could have just have multiple columns in a row.

**Comment by skube** [comment]
I must be doing something wrong because when I repeat the placeholders on a slide, I get exact exactly that, multiple identical merges per slide and a new slide for each row of data. 

<img width="1206" alt="screen shot 2017-09-08 at 10 15 56 am" src="https://user-images.githubusercontent.com/146396/30215679-c78dcaca-947e-11e7-9c65-cb024d401efa.png">

**Comment by brucemcpherson** [comment]
And that isn't what you want?

**Comment by skube** [comment]
Nope. The final merged result should have **all** the rows on **one** slide. 

So for the image above, the merged Slide would have:
> My sub-task **one**, My sub-task **two**, My sub-task **three**, etc.

Not:
> My sub-task _one_, My sub-task _one_, My sub-task _one_, etc.

**Comment by brucemcpherson** [comment]
So why can't you just have one row, with columns named subtask1, subtask2 etc... btw I couldn't see any images if you sent any.

**Comment by skube** [comment]
I _could_ have one long row, but that would mean creating unique headings in both Sheets and Slides. Besides which, it isn't a very good way to organize data in Sheets and makes exporting to CSV an issue. 

I'm not sure why you can't see the images I've attached.  Are you viewing this on github.com or in an email thread?

**Comment by brucemcpherson** [comment]
I got them now. No you can't do that, but you could maintain data in one sheet in use transpose formula to turn rows in to columns in the data.

Alternatively you could use your sheet as a globals sheet, which is organized rowwise, and your data sheet would just have a single dummy row.

**Comment by skube** [comment]
I see. Appreciate the suggestions however they each seem arduous and time-consuming thus defeating the whole point. Especially when dealing with multiple placeholders.

I guess there's no easy way to extend SlidesMerge to accommodate such functionality?
